src/page/Timeline.js

```diff
@@ -0,0 +1,336 @@
+                                    "name": "Ivan"
+                                },
+                                "action": "add",
+                                "description": "Adding evil Ivan",
+                            },
+                            {
+                                "subject": {
+                                    "type": "edge",
+                                    "edge": [0, 1], // from, to
+                                    "name": "Friendship"
+                                },
+                                "action": "add",
+                                "description": "description 3c3 ",
+                            }
+                        ]
+                    },{
+                        "vertices": [[]],
+                        "edges": [[]],
+                        "id": 8,
+                        "changelog": 
+                        [
+                            {
+                                "subject": {
+                                    "type": "vertex",
+                                    "idx": 1, // idx
+                                    "name": "Ivan"
+                                },
+                                "action": "add",
+                                "description": "Adding more Ivan",
+                            },
+                            {
+                                "subject": {
+                                    "type": "edge",
+                                    "edge": [0, 1], // from, to
+                                    "name": "Friendship"
+                                },
+                                "action": "add",
+                                "description": "description 11",
+                            }
+                        ]
+                    },{
+                        "vertices": [[]],
+                        "edges": [[]],
+                        "id": 9,
+                        "changelog": 
+                        [
+                            {
+                                "subject": {
+                                    "type": "vertex",
+                                    "idx": 1, // idx
+                                    "name": "Ivan"
+                                },
+                                "action": "add",
+                                "description": "Adding anotherIvan",
+                            },
+                            {
+                                "subject": {
+                                    "type": "edge",
+                                    "edge": [0, 1], // from, to
+                                    "name": "Friendship"
+                                },
+                                "action": "add",
+                                "description": "description 12",
+                            }
+                        ]
+                    },
+                ]
+            };
```

Comment:
[MAJOR] This should be extracted into a file of its own for the time being. It pollutes the code base.